Add H-K lettered article stubs from glossary (#3828)
ppy/osu-wiki: wiki/Glossary/Katu/en.md

---
tags:
  - 100
---

# Katu

> A **katu(喝)**, or *Beat!*, is when you complete a [comboset](/wiki/Beatmapping/Combo/) without achieving the highest level of [accuracy](/wiki/Accuracy/). If instead the comboset was done perfectly a Geki(激) would be achieved.

Reviewer: ```suggestion
A **Katu (喝)**, or *Beat!*, is a score that results from the completion of a [comboset](/wiki/Beatmapping/Combo) without achieving the highest level of [accuracy](/wiki/Accuracy). If the comboset was done perfectly, a **Geki (激) ** would appear.
```
